### DISCUSSION TITLE
Edge function pricing

### DISCUSSION THREAD

**Discussion by zzza** [supabase/supabase#31011] [Questions] [answered]
Other than number of invocations, are there any other factors that can affect the pricing of edge functions? e.g. how long they run for?

**Comment by GaryAustin1** [comment, marked as answer]
Not currently. Just invocations with the fixed limits.

**Comment by zzza** [comment]
Fantastic, thanks Gary!

**Comment by FBSODigital** [comment]
I'm getting error messages: "unexpected create function status 403: {"message":"Max number of functions reached for project"}" I don't seem to have that many functions. What is the max number of functions in Supabase and if I need more how would I get them?  I don't see any functional limits on this page nor do I see limits in the pricing packages. https://supabase.com/docs/guides/functions/limits

**Reply by GaryAustin1** [reply]
https://github.com/orgs/supabase/discussions/34530#discussioncomment-12662332
